src/Core/Managed/Shared/TelemetryClient.cs

```diff
@@ -293,11 +293,10 @@ public void Track(ITelemetry telemetry)
             if (this.IsEnabled())
             {
                 this.Initialize(telemetry);
-                
-                this.WriteTelemetryToDebugOutput(telemetry);
```

Comment:
Even if telemetry was sampled Visual Studio diagnostics hub will still need to show this telemetry item. Perhaps we can somehow indicate that telemetry was sampled out by setting sampleRate to zero. 

So I'd recommend just to move this call couple lines to the bottom right after the call to processors